### PR TITLE
DM-45886: Make PSF plotting work for LSSTCam

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

File: python/lsst/summit/extras/plotting/psfPlotting.py [hunk 1]
```diff
@@ -475,51 +480,53 @@ def makeAzElPlot(
...
-                det.getName(),
-                rotation_mode="anchor",
-                rotation=np.rad2deg(-rtp) - 90,
-                horizontalalignment="left",
-                verticalalignment="top",
-                color="k",
-                fontsize=6,
-                zorder=20,
-            )
+    if oneRaftOnly:  # no labels on full camera plots as it's overwhelming
```
Comment (erykoff) [inline python/lsst/summit/extras/plotting/psfPlotting.py:496]: I would rephrase this comment "Only plot labels on single-raft cameras; otherwise it is overwhelming"

File: python/lsst/summit/extras/plotting/psfPlotting.py [hunk 2]
```diff
@@ -448,6 +448,11 @@ def makeAzElPlot(
     saveAs : `str`, optional
         The file path to save the figure.
     """
+    oneRaftOnly = camera.getName() in ["LSSTComCam", "LSSTComCamSim", "TS8"]
+    cameraFactor = 4.5  # I think this is roughly right for plotting - diameter is 5x but we need less border
```
Comment (erykoff) [inline python/lsst/summit/extras/plotting/psfPlotting.py:452]: Maybe call this `fullCameraFactor`?

**Review by erykoff — APPROVED** [review]

File: python/lsst/summit/extras/plotting/psfPlotting.py (hunk 1 above)
Comment (mfisherlevine) [reply python/lsst/summit/extras/plotting/psfPlotting.py:496]: Sure.

File: python/lsst/summit/extras/plotting/psfPlotting.py (hunk 2 above)
Comment (mfisherlevine) [reply python/lsst/summit/extras/plotting/psfPlotting.py:452]: Sure.
